src/AdafruitIO_Definitions.h

```diff
@@ -124,11 +124,15 @@ class AdafruitIOGroupCallback {
                                                                 ///< SSL
                                                                 ///< Fingerprint
 
-#define AIO_FEED_NAME_LENGTH 20 ///< Maximum length of an Adafruit IO Feed name
+#define AIO_FEED_NAME_LENGTH                                                   \
+  258 ///< Maximum length of an Adafruit IO Feed    \
+                                  ///< Name; 128 + 1 + 128 for the group, a dot \
+                                  ///< , and actual feed name.
 #define AIO_DATA_LENGTH                                                        \
   45 ///< Maximum length of data sent/recieved from Adafruit IO
 #define AIO_CSV_LENGTH                                                         \
-  150 ///< Maximum comma-separated-value length from Adafruit IO
```

Comment:
Why `+4`?

Reply:
https://github.com/adafruit/Adafruit_IO_Arduino/issues/127#issuecomment-654242578

Reply:
@blaskovicz - ok, rest looks good. I'll test by making some long feed names and testing on physical hardware tomorrow.

Reply:
@brentru any luck?